src/studio/src/designer/frontend/app-development/features/iFrame/iFrameComponent.tsx

```diff
@@ -34,13 +34,16 @@ export function IFrameComponent({
   classes,
   iframeEndingUrl,
 }: IIFrameComponentProvidedProps) {
-  const { org, app } = window as Window as IAltinnWindow;
+  const { org, app } = useParams();
 
   const url = `${window.location.origin}/designer/${org}/${app}/${iframeEndingUrl}`;
   return (
     <div className={classes.mainLayout}>
       {/* eslint-disable-next-line jsx-a11y/iframe-has-title */}
-      <iframe className={classes.iFrameLayout} src={url} />
+      <iframe
```

Comment:
i think there might be an issue with loosing the `org` `app` values in a iframe, dont really know why we used it in the first place.

Reply:
What is the iframe-thing doing? I guess that is something that we at some point could remove from the codebase?

Reply:
Honestly can't come up with other areas that it is practically used today.